amundsen_application/base/examples/example_redash_preview_client.py

```diff
@@ -0,0 +1,118 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+
+from typing import Dict, Optional
+
+from amundsen_application.base.base_redash_preview_client import BaseRedashPreviewClient
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+# Redash natively runs on port 5000, the same port as Amundsen.
+# Make sure to update the running port to match your deployment!
+DEFAULT_URL = 'http://localhost:5010'
+
+
+SOURCE_DB_QUERY_MAP = {
+    'snowflake.ca_covid': 1,
```

Comment:
Yes and no. Some of the other clients use upstream confs for auth but not for business logic-specific code, superset actually defines a similar dictionary for the database mapping.


[Superset](https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/base/examples/example_superset_preview_client.py#L13) and [bigquery](https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/base/examples/example_bigquery_preview_client.py#L22) have configs in the code but [dremio](https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/base/examples/example_dremio_preview_client.py#L43) uses upstream confs. 

Since clients _likely_ to implement their own preview class which will require a high touch, IMHO having this mapping directly in the file is OK but I would be happy to explore a more configuration-driven approach to having some or all of these preview clients being turn-key.

Reply:
resolved synchronously, makes sense to include